### PR TITLE
feat: Additional tags for security group

### PR DESCRIPTION
**Pull request by DJHig**
## Description
Provide an additional set of tags for the security group created

## Motivation and Context
Needed for enterprise security audits

## Breaking Changes
None

## How Has This Been Tested?
- [x] I have tested and validated these changes using one or more of the provided `examples/*` projects
- Verified that providing this value does tag the security group correctly.
- Works like cluster_tags

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -268,6 +268,12 @@ variable "cluster_tags" {
   default     = {}
 }
 
+variable "security_group_tags" {
+  description = "A map of tags to add to created security group, in addition to the other tags."
```
Comment (bryantbiggs) [inline variables.tf:272]:
```suggestion
  description = "Additional tags for the security group"
```

**Comment by bryantbiggs** [comment]
@DJHig left one minor comment, and then if you could change your PR title to something like `feat:  additional tags for security group` then we should be all set - thank you!

**Comment by DJHig** [comment]
Done. Thank you. 
I am not sure why the pre-commit check is failing.

**Comment by bryantbiggs** [comment]
@DJHig it doesn't look like the README is getting updated when you run `pre-commit run -a`

```
-| security\_group\_tags | A map of tags to add to created security group, in addition to the other tags | `map(string)` | `{}` | no |
+| security\_group\_tags | Additional tags for the security group | `map(string)` | `{}` | no |
```

**Comment by DJHig** [comment]
Weird it did not show the updates. 
Manually applied the change to readme.

**Review by bryantbiggs — APPROVED** [review]
@antonbabenko 👍🏼

**Comment by antonbabenko** [comment]
Thank you, @DJHig and @bryantbiggs !

[v3.13.0](https://github.com/terraform-aws-modules/terraform-aws-rds-aurora/releases/v3.13.0) has been just released.

**Pull request merged**